views/harvestlogbook.ejs

```diff
@@ -107,7 +107,7 @@
                                             <%}else{ %>
                                                 <a onclick="addToBlockchain(this)" class="btn btn-block btn-xs btn-success blockchain addHarvestToBlockchainBtn" id="<%= (" harvest_add_blockchain_ " + data[i].harvest_logid) %>" data-harvest_logid="<%= data[i].harvest_logid%>" data-harvest_suppliershortcode="<%=data[i].harvest_supplierShortcode%>"
                                                     data-harvest_suppliername="<%=data[i].harvest_supplierName%>" data-harvest_farmername="<%=data[i].harvest_farmerName%>" data-harvest_supplieraddress="<%=data[i].harvest_supplierAddress%>" data-harvest_producename="<%=data[i].harvest_produceName%>"
-                                                    data-harvest_photohash="<%=data[i].harvest_photoHash%>" data-harvest_timestamp="<%=data[i].harvest_TimeStamp%>" data-harvest_capturetime="<%=data[i].harvest_CaptureTime%>" data-harvest_description="<%=data[i].harvest_Description%>"
+                                                    data-harvest_photohash="<%=data[i].harvest_image_url%>" data-harvest_timestamp="<%=data[i].harvest_TimeStamp%>" data-harvest_capturetime="<%=data[i].harvest_CaptureTime%>" data-harvest_description="<%=data[i].harvest_Description%>"
```

Comment:
@rideam Have you tested that when you click on Add to Blockchain, the image and not the image_url is correctly hashed in the inline addToBlockchain JS function i.e. `                let photoHash = sha256(harvestPhotoHash);`?

The quick way to check is that if you do a SHA256 hash of the file url from the console, this should be different from the SHA256 that is stored in the blockchain record.

Reply:
@jajukajulz with the above change the url and not the image is being hashed ,, trying to fetch the image before the call to the sha256 funtion is causing cors issues

Reply:
this is fixed now ,, implemented a workaround to get the hash via the route `getimagehash` in harvest.js and not hash the file in ejs